shell/src/main/java/org/crsh/util/Utils.java

```diff
@@ -47,7 +47,11 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-public class Utils {
+public final class Utils {
+	
+  private Utils(){
+	  throw new AssertionError("Must not instantiate this class");
```

Comment:
If this is really necessary, perhaps it should throw an IllegalStateException, instead.